tests/expectations/core/test_expect_column_values_to_be_string_integers_monotonically_increasing.py

```diff
@@ -0,0 +1,269 @@
+#         spark_session, basic_spark_df_execution_engine,
+# ):
+#     with pytest.raises(MetricResolutionError):
+#
+#         df = pd.DataFrame(
+#             {
+#                 "a": [
+#                     "1",
+#                     "2",
+#                     "3",
+#                     "3",
+#                     "0",
+#                     "6",
+#                     "2021-05-01",
+#                     "test",
+#                     8,
+#                     9
+#                 ]
+#             }
+#         )
+#
+#         validator = build_spark_validator_with_data(df, spark_session)
+#
+#         result = validator.expect_column_values_to_be_string_integers_monotonically_increasing(column='a')
+#
+#
+# def test_sa_expect_column_values_to_be_string_integers_monotonically_increasing_success(
+#         test_backends
+# ):
+#     df = pd.DataFrame(
+#         {
+#             "a": [
+#                 "0",
+#                 "1",
+#                 "2",
+#                 "3",
+#                 "3",
+#                 "9",
+#                 "11"
+#             ]
+#         }
+#     )
+#
+#     validator = build_sa_validator_with_data(df=df, sa_engine_name='postgresql')
+#
+#     result = validator.expect_column_values_to_be_string_integers_monotonically_increasing(column='a')
+#
+#     assert result == ExpectationValidationResult(
+#         exception_info={
+#             "raised_exception": False,
+#             "exception_traceback": None,
+#             "exception_message": None,
+#         },
+#         expectation_config={
+#             "expectation_type": "expect_column_values_to_be_string_integers_monotonically_increasing",
+#             "kwargs": {
+#                 "column": "a",
+#                 "batch_id": "57175eeb4a8baa7ae63f44c6540eb559",
+#             },
+#             "meta": {},
+#             "ge_cloud_id": None,
+#         },
+#         meta={},
+#         result={
+#             "observed_value": [
+#                 [
+#                     True
+#                 ],
+#                 [
+#                     6
+#                 ]
+#             ]
+#         },
+#         success=True
+#     )
+#
+#
+# def test_sa_expect_column_values_to_be_string_integers_monotonically_increasing_failure(
+#         test_backends
+# ):
+#     with pytest.raises(MetricResolutionError):
+#
+#         df = pd.DataFrame(
+#             {
+#                 "a": [
+#                     "1",
+#                     "2",
+#                     "3",
+#                     "3",
+#                     "0",
+#                     "6",
+#                     "2021-05-01",
+#                     "test",
+#                     8,
+#                     9
+#                 ]
+#             }
+#         )
+#
+#         validator = build_sa_validator_with_data(df=df, sa_engine_name='postgresql')
+#
+#         result = validator.expect_column_values_to_be_string_integers_monotonically_increasing(column='a')
```

Comment:
If you run `black .` on your GE repo it will do the linting for you and add a line here. See: https://docs.greatexpectations.io/docs/contributing/style_guides/code_style#code

Reply:
Typically running `black .` and `isort .` is good enough for me but you sometimes might need to run other commands in that link to get past the `lint lint` step of the CI/CD checks.